gaps/gap-16_golem_compose/gap-16_golem_compose.md

```diff
@@ -0,0 +1,177 @@
+---
+gap: GAP-16
+title: Multi-service application deployment framework for Golem Network
+description: A solution for defining, deploying and managing apps based on multiple services operating within one or more Golem VPNs
+author: Jakub Mazurek (@zakaprov) <jakub.mazurek@golem.network>
+status: Draft
+type: Feature
+requires (*optional): <GAP number(s)>
+replaces (*optional): <GAP number(s)>
+---
+
+## Abstract
+This GAP introduces a proposal for a language-agnostic layer of abstraction over the Golem VPN and its high-level APIs (`yapapi`, `yajsapi`).
```

Comment:
I think it should be possible to also use this descriptor for services that _don't_ require VPN